### ISSUE TITLE
bug: Menu (Platform): menu and menu items missing tooltip

### ISSUE THREAD

**Issue by Seamoo13** [SAP/fundamental-ngx#4119]
#### Is this a bug, enhancement, or feature request?
bug
#### Briefly describe your proposal.
menu and menu items should have tooltip

![2020-12-16_1](https://user-images.githubusercontent.com/47522152/102325950-72dd2900-3f8c-11eb-9f73-c2a7cce3d1ec.png)

split from: https://github.com/SAP/fundamental-ngx/issues/4038
